<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Search</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>971</width>
      <height>361</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="searchStartButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search From Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>971</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search phrase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadMoreButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>971</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load more</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchNextButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Next</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLemmatization">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lemmatization</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonStemming">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stemming</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNone">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
